### PR TITLE
Add policy pages

### LINKED ISSUE #1722: Update policy pages
## The list of requirements

- Adds a privacy policy, cookie policy and imprint page to the app

## Links

https://www.notion.so/gnosis-safe/Terms-policy-update-f3827a6f724340deae9554d24afe8c29

### PR DESCRIPTION
**Pull request by usame-algan**
## What it solves

Resolves #1722 

## How this PR fixes it

- Adds a new page for Privacy policy, Cookie policy, Terms and Imprint
- Links those pages in the footer
- Displays the footer on those pages
- Hides the sidebar on those pages

## How to test it

1. Open the Safe
2. Go to Settings to see the Footer
3. Click on any of the three links to see the page
4. Compare to https://www.notion.so/gnosis-safe/Terms-policy-update-f3827a6f724340deae9554d24afe8c29
6. Create or Load a new safe and click on terms of use or privacy policy and observe seeing the new pages

## Screenshots

Screenshot 2023-03-03 at 11 06 23

### CODE REVIEW

**Review by katspaugh — COMMENTED** [review]
Why do you want to have them in this repo and not just link to the homepage?

File: src/components/imprint/index.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        ℅ Full Node
+        <br />
+        Skalitzer Str. 85-86
+        <br />
+        10997 Berlin Germany
+      </Typography>
+      <Typography mb={4}>
+        Managing directors: Richard Meißner, Tobias Schubotz
+        <br />
+        Contact: info@cc0x.dev
```
Comment (iamacook) [inline src/components/imprint/index.tsx:26]: We should probably make this a `mailto` link for consistency with the privacy page.

File: src/components/new-safe/create/steps/SetNameStep/index.tsx [hunk 2]
```diff
@@ -13,7 +13,10 @@ import useIsWrongChain from '@/hooks/useIsWrongChain'
 import NetworkWarning from '@/components/new-safe/create/NetworkWarning'
 import NameInput from '@/components/common/NameInput'
 import { CREATE_SAFE_EVENTS, trackEvent } from '@/services/analytics'
-import ExternalLink from '@/components/common/ExternalLink'
+import { AppRoutes } from '@/config/routes'
+import MUILink from '@mui/material/Link'
+import Link from 'next/link'
+import React from 'react'
```
Comment (iamacook) [inline src/components/new-safe/create/steps/SetNameStep/index.tsx:19]:
```suggestion
```

File: src/components/privacy/index.tsx [hunk 3]
```diff
@@ -0,0 +1,886 @@
...
+
+const SafePrivacyPolicy = () => {
+  return (
+    <div>
+      <h1>Privacy Policy</h1>
+      <p>Last updated on March&nbsp;2023.</p>
+      <p>
+        Your privacy is important to us. It is our policy to respect your privacy and comply with any applicable law and
+        regulation regarding any personal information we may collect about you, including across our website,{' '}
+        <a href="https://app.safe.global">https://app.safe.global</a>
```
Comment (iamacook) [inline src/components/privacy/index.tsx:13]: There are a few anchor elements on this page. Should they be `MUILink`?

File: src/components/privacy/index.tsx [hunk 4]
```diff
@@ -0,0 +1,886 @@
...
+        <br />
+        10555 Berlin
+        <br />
+        Germany
+        <br />
+        Phone: 030/138 89-0
+      </p>
+      <p>
+        <Link
+          href="https://www.google.com/url?q=https://www.datenschutz-berlin.de&amp;sa=D&amp;source=editors&amp;ust=1677670462021305&amp;usg=AOvVaw1sgbmKiiYL0n7tPM5It81-"
```
Comment (iamacook) [inline src/components/privacy/index.tsx:822]: The AMP needs removing from this link.

File: src/components/privacy/index.tsx (hunk 3 above)
Comment (usame-algan) [reply src/components/privacy/index.tsx]: Changed it

File: src/components/imprint/index.tsx (hunk 1 above)
Comment (usame-algan) [reply src/components/imprint/index.tsx]: done!

File: src/components/privacy/index.tsx (hunk 4 above)
Comment (usame-algan) [reply src/components/privacy/index.tsx]: Good catch, I removed the AMP link

File: src/components/imprint/index.tsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        ℅ Full Node
+        <br />
+        Skalitzer Str. 85-86
+        <br />
+        10997 Berlin Germany
+      </Typography>
+      <Typography mb={4}>
+        Managing directors: Richard Meißner, Tobias Schubotz
+        <br />
+        Contact: <a href="mailto:info@cc0x.dev">info@cc0x.dev</a>
```
Comment (iamacook) [inline src/components/imprint/index.tsx:26]:
```suggestion
        Contact:{' '}
        <Link href="mailto:info@cc0x.dev" passHref>
          <MUILink>info@cc0x.dev</MUILink>
        </Link>
```

**Review by iamacook — APPROVED** [review]

**Comment by usame-algan** [comment]
Closing this in favor of #1731

**Pull request closed without merging**